src/robot_gnss/skeleton.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+# СКЕЛЕТ ОСНОВНОЙ ПРОГРАММЫ
+
+import rospy
+from numpy import nan, isnan
+from sensor_msgs.msg import NavSatFix
+import gnss_math as gm
+
+# ====
+# УГОЛ
+# ====
+latitude = nan
+longitude = nan
+
+
+def nav_cb(nsf):
+    latitude = nsf.latitude
+    longitude = nsf.longitude
+
+
+rospy.init_node('robot', anonymous=True)
+nav = rospy.Subscriber('fix', NavSatFix, nav_cb, queue_size=10)
+
+# ===============
+# ПОЛУЧЕНИЕ ТОЧЕК
+# ===============
+# todo: получаем точки
+point_list = [[100, 100][200, 200][300, 300][400, 400]]  # загруженные точки
```

Comment:
What are these points? Are these trajectory waypoints? Are they in local coordinates? For the future, the trajectory points will be read from the .json file.

Reply:
This is a random value of latitude and longitude. This code is intended to explain to another person what his program should look like (this is the skeleton)